primitives/npos-elections/src/mock.rs

```diff
@@ -314,7 +309,7 @@ pub fn check_assignments_sum<T: PerThing>(assignments: Vec<Assignment<AccountId,
 	for Assignment { distribution, .. } in assignments {
 		let mut sum: u128 = Zero::zero();
 		distribution.iter().for_each(|(_, p)| sum += p.deconstruct().saturated_into::<u128>());
-		assert_eq_error_rate!(sum, T::ACCURACY.saturated_into::<u128>(), 1);
+		assert_eq!(sum, T::ACCURACY.saturated_into::<u128>(), "Assignment ratio sum is not 100%");
```

Comment:
Why should we use `assert_eq`?

Reply:
Tests are failing otherwise. That syntax is actually used in the target commit